src/main/java/Duke.java

```diff
@@ -1,10 +1,133 @@
+        }
+    }
+    public static void greet(String logo) {
+        System.out.println("\n" + logo);
+        line();
+        System.out.println("\tHello Sir,\n" + "\tWhat can I do for you?\n"); // greet
+        line();
+    }
+    public static void exit() {
+        System.out.println("Goodbye Sir. I shall be at your service again whenever you require\n"); //exit message
+        line();
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String logo = "                       ./((((((((((((((/.                    \n" +
+                "                  /(((/.  .,*////*,.   *((((                \n" +
+                "               (#(*  (.       ,((*.     #. ,#(#             \n" +
+                "            .((/ .*    (*     ,   ./((((,  ,, *((,          \n" +
+                "           ((/ /   .(        (##        (((,  ( *((.        \n" +
+                "         .(( ..  .,         #####         .((,  , #(*       \n" +
+                "        .(( ,   #          ####( ,,         /((  / ((,      \n" +
+                "        ((  ,  (         ,####* /##(         *(( .. ((      \n" +
+                "        (( /  ..        (####,  #####         #(. # ((.     \n" +
+                "        /              %####     #####        #(.    #      \n" +
+                "        ,  *   ,                  /####,      ((  (  /      \n" +
+                "         (  *  /    ,###################(    ((* ,  ,       \n" +
+                "         .   ,  *  /######################  ((* ..  ,       \n" +
+                "           ,  #   (                       (((  (   .        \n" +
+                "            (   (   ,/                 (((/  *.  #          \n" +
+                "              (   *,    ,(*.  ,/((#((((,  ./   #            \n" +
+                "                 #    (*              ,(    (.              \n" +
+                "                    ,#                  (*                  \n" +
+                "                           .,.   ,.                         \n" +
+                "                                                            \n" +
+                "                                                            \n" +
+                "      ,/,     ,//      //////*    //      */. */.    */////*\n" +
+                "      /@*    (@#@&            @@  .@&    %@*  #@*  &@       \n" +
+                "      /@*   #@* .@@    @@@@@@@@*   .@@  @@,   #@*  .@@@@@@@.\n" +
+                "      @@.  %@,    @@   @@   #@(      @@&@.    #@*         @&\n" +
+                " */**.    */.      /*  **    ./*      */      ,*.  ,******  ";
+        greet(logo);
+        Tasks user = new Tasks();
```

Comment:
A different variable name can be used to show tasks is a plural collection of the user's tasks.